composer.json

```diff
@@ -17,8 +17,8 @@
     ],
     "require": {
         "php": "^5.6 || ^7.0",
-        "doctrine/phpcr-bundle": "^1.1",
-        "doctrine/phpcr-odm": "^1.1 || ^2.0",
+        "doctrine/phpcr-bundle": "^1.3",
+        "doctrine/phpcr-odm": "^1.4 || ^2.0",
```

Comment:
remember that for this repo, master is the current stable and 2.0 is the upcoming new version. we should not bump those versions in a patch release if not absolutely necessary. when we use the jackalope-jackrabbit that has repository setup right, lets see if we can go down on these again.